lib/aerosol/aws_model.rb

```diff
@@ -5,6 +5,7 @@ def self.included(base)
       extend ClassMethods
 
       attr_accessor :from_aws
+      attr_accessor :launch_template
```

Comment:
ditto this since this means a launch config has a property called launch template, right?

Reply:
Yeah, this could move to the classes that actually have launch templates. Should just be AutoScaling and Instance